code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -425,8 +425,8 @@ Contains:
 	heat_protection = NONE
 	flash_protect = 0
 	bang_protect = 0
-	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEHAIR|HIDEFACIALHAIR
-	clothing_flags = STOPSPRESSUREDAMAGE | SHOWEROKAY | SNUG_FIT
```

Comment:
You have changed other aspects that might still have flags. If there is a showerokay flag or one of a similar style please re-add it.

Reply:
I may be stupid, will fix it now